kratos/python_scripts/from_json_check_result_process.py

```diff
@@ -24,6 +24,47 @@ class FromJsonCheckResultProcess(KratosMultiphysics.Process, KratosUnittest.Test
     settings -- Kratos parameters containing solver settings.
     """
 
+    def __init__(self, model, params):
+        """ The default constructor of the class
+
+        Keyword arguments:
+        self -- It signifies an instance of a class.
+        model -- the model contaning the model_parts
+        settings -- Kratos parameters containing solver settings.
+        """
+        KratosMultiphysics.Process.__init__(self)
+        self.process = KratosMultiphysics.FromJSONCheckResultProcess(model, params)
+
+    def ExecuteInitialize(self):
+        """ This method is executed at the begining to initialize the process
+
+        Keyword arguments:
+        self -- It signifies an instance of a class.
+        """
+        self.process.ExecuteInitialize()
+
+    def ExecuteFinalizeSolutionStep(self):
+        """ This method is executed in order to finalize the current step
+
+        Here the dictionary containing the solution is filled
+
+        Keyword arguments:
+        self -- It signifies an instance of a class.
+        """
+        self.process.ExecuteFinalizeSolutionStep()
+        self.assertTrue(self.process.IsCorrectResult(), "Results do not coincide with the JSON reference")
+
+class LegacyFromJsonCheckResultProcess(KratosMultiphysics.Process, KratosUnittest.TestCase):
```

Comment:
The old process is preserved in LegacyFromJsonCheckResultProcess

Reply:
If the behavior is the same, why do we need to keep the old Python version? (I understand that the `Legacy...` implements the old Python version.

Reply:
in ParticleMechanicsApplication/python_scripts/particle_from_json_check_result_process.py , they created their own by deriving. So this code should be ported to cpp

Reply:
Basically I keep it in order to not break the PartcileMechanicsApp as @pablobecker says